.github/workflows/release-docs-and-schema.yml

```diff
@@ -37,12 +40,6 @@ jobs:
       - uses: actions/setup-node@v2.5.1
         with:
           registry-url: 'https://registry.npmjs.org'
-      - name: Publish to NPM registry
```

Comment:
This was removed because it's already been released once on the trigger to the `push` event